assets/wizards/readerRevenue/views/emails/index.js

```diff
@@ -87,6 +87,13 @@ const Emails = () => {
 		<>
 			{ emails.map( email => {
 				const isActive = email.status === 'publish';
+				const notification =
+					email.type === 'cancellation'
+						? __( 'This email is not active.', 'newspack-plugin' )
+						: __(
+								'This email is not active. The default receipt will be used.',
+								'newspack-plugin'
+						  );
```

Comment:
This should rather be the other way around – if the type is receipt, use the message about the default receipt. Otherwise, use the generic message.

Reply:
> This should rather be the other way around – if the type is receipt, use the message about the default receipt. Otherwise, use the generic message.

Do you mean this should be changed to:

```
				const notification =
					email.type === 'receipt'
						? __(
								'This email is not active. The default receipt will be used.',
								'newspack-plugin'
						  )
                        : __( 'This email is not active.', 'newspack-plugin' )
```

If so, this will have the same result as the current changes. On my end, it looks like the messaging is correct on the frontend as well:

![Screenshot 2024-04-17 at 09 35 47](https://github.com/Automattic/newspack-plugin/assets/17905991/a6b3833c-6e06-4a15-9f34-f8852b3a0e0a)

Or did you mean something else?

Reply:
That's what I meant. The result is the same, but the code is more logical.

Reply:
I see. Happy to make the change if you feel that way! Done in https://github.com/Automattic/newspack-plugin/pull/3056/commits/052255fd74e40b7ef2183d8c0fa4be2c29d1919f

Reply:
It's not about feelings, it's logic & readability:

```
type === 'receipt' ? notification_about_receipt : generic_notification
```

instead of 

```
type === 'cancellation' ?  generic_notification : notification_about_receipt
```

As long as type can only be `'receipt'` or `'cancellation'` the former version seems fine, but it's not a good assumption to build on (there will be other types).